app/src/views/CountryNsOverviewContextAndStructure/NationalSocietyLocalUnits/LocalUnitValidateButton/index.tsx

```diff
@@ -91,31 +35,32 @@ function LocalUnitValidateButton(props: Props) {
 
             return strings.localUnitsValidate;
         },
-        [isValidated, readOnly, strings],
+        [isValidated, strings, readOnly],
     );
 
     return (
-        <ConfirmButton
+        <Button
             className={_cs(isValidated
                 ? styles.localUnitValidatedButton
                 : styles.localUnitValidateButton)}
-            // NOTE sending an empty post request to validate the local unit
-            name={null}
+            name={undefined}
+            onClick={onClick}
```

Comment:
```suggestion
            onClick={onClick}
```
use better naming convention.